### ISSUE TITLE
Change source strings with `->` to `→`?

### ISSUE THREAD

**Issue by ovari** [moneymanagerex/moneymanagerex#5462]
Suggestion:
Change en source strings with `->` to `→`?

https://crowdin.com/translate/moneymanagerex/470/en-hu?filter=basic&value=0#q=-%3E

What do you think?

Thank you

**Comment by ovari** [comment]
`->` not present https://crowdin.com/translate/moneymanagerex/all/en-hu?filter=basic&value=0#q=-%3E
`→` present https://crowdin.com/translate/moneymanagerex/all/en-hu?filter=basic&value=0#q=%E2%86%92

Thank you

**Comment by ovari** [comment]
`WebApp   API version: %s`

Should the 3 consecutive spaces `   ` be changed to just 1 space ` ` in https://github.com/vomikan/moneymanagerex/commit/2058792b6063c0f7a67d4396ceee0932ec4b41ed#diff-d724d5a0bd2ee9450a167a257401b31cd5ce2c38d3e1e3c1636210cb148f6d42R7609

**Comment by ovari** [comment]
Should `Optimize` be changed to `Optimize Database`? In the string `We suggest a database optimization under Tools → Database → Optimize`.

https://github.com/vomikan/moneymanagerex/commit/25042d616fe7632b4fbbdefda8d453cae57272c9#diff-5b877b5db74e6dc53e42f9ff54fb4d509f7198867ac091e3e495db5f6504994cR121

Perhaps the string could be written in third-person to be:
`Database optimization recommended using the Tools → Database → Optimize Database menu`

Unless the `Optimize Database…` menu is renamed to `Optimize…` as it is in the `Database` menu already.
If so, the other items could be renamed from `Debug Database…` to `Debug…`

**Comment by ovari** [comment]
Should `Relocation of` be changed to `Relocate`?

`Tools → Relocation of → Categories` to `Tools → Relocate → Categories`
`Tools → Relocation of → Payees` to `Tools → Relocate → Payees`
`Re&location of` to `Re&locate`

https://github.com/vomikan/moneymanagerex/commit/25042d616fe7632b4fbbdefda8d453cae57272c9#diff-5899dbffbc1b317eb60d03c7618f148c1720abcbfb4598f9d66746ced0adcd25R476

https://crowdin.com/translate/moneymanagerex/all/en-hu?filter=basic&value=0#q=relocation
https://crowdin.com/translate/moneymanagerex/all/en-hu?filter=basic&value=0#q=re&location

**Comment by ovari** [comment]
Should `Switch Application &Language` be changed to `User Interface &Language`?

https://www.microsoft.com/en-us/language/Search?&searchTerm=user%20interface%20language&langID=382&Source=true&productid=undefined

It seems better than the current string or `Change application language` as users can look in the menu to see the current language without changing or switching the language.
https://www.microsoft.com/en-us/language/Search?&searchTerm=application%20language&langID=382&Source=true&productid=undefined

What do you think?

Thank you

**Comment by rmelillo76** [comment]
Arrows are not displaying correctly after this change, refer to a couple of examples below. I am using Windows 10:

![image](https://user-images.githubusercontent.com/95182117/210192368-e9adc62e-2815-42a9-847c-34939942ee34.png)

![image](https://user-images.githubusercontent.com/95182117/210192379-ce75e025-282a-4e97-aa79-9fdf5a5269e9.png)

**Comment by ovari** [comment]
> Arrows are not displaying correctly after this change, refer to a couple of examples below. I am using Windows 10:

Displays correctly with Linux Mint 21.1 Cinnamon in the Hungarian (magyar) locale.

![Arrows display correctly with GNU/Linux operating system in the Hungarian (magyar) locale](https://user-images.githubusercontent.com/17465872/210193942-55f54867-6108-4a5d-a7b5-df221134377c.png)

Are the source strings in [ASCII](https://en.wikipedia.org/wiki/ASCII) format?

Hopefully the strings are in [UTF-8](https://en.wikipedia.org/wiki/UTF-8) format.

**Comment by ovari** [comment]
> Arrows are not displaying correctly after this change, refer to a couple of examples below. I am using Windows 10:

English works too with:
* Linux Mint 21.1 Cinnamon
* Money Manager Ex
Version: 1.6.3-Beta.1 64-bit
• Build on Jan 2 2023 03:26:00
• Database version: 17 (aes128cbc)

![image](https://user-images.githubusercontent.com/17465872/210217589-43c68b52-170b-4fc6-bc90-3dc7eafea3e6.png)

@rmelillo76 can you please paste the `MMEX About tab` from the Help → About menu? Perhaps this can help the developers.

![MMEX About tab](https://user-images.githubusercontent.com/17465872/210218039-0c447fec-f610-4e14-9d19-99ec3d0358fc.png)

**Comment by ovari** [comment]
Can the `->` in the `Privacy` tab in the `Help`, `About` menu please be updated?

![Privacy tab](https://user-images.githubusercontent.com/17465872/210218875-0e2c51bd-e4eb-4907-bd13-8721b338e01a.png)

Thank you

**Comment by vomikan** [comment]
> Arrows are not displaying correctly after this change, refer to a couple of examples below. I am using Windows 10:

Could you check the following settins of your OS:
![image](https://user-images.githubusercontent.com/6836805/210228512-094e7316-44b0-433d-83f4-674739d43724.png)

**Comment by rmelillo76** [comment]
> > Arrows are not displaying correctly after this change, refer to a couple of examples below. I am using Windows 10:
> 
> Could you check the following settins of your OS: ![image](https://user-images.githubusercontent.com/6836805/210228512-094e7316-44b0-433d-83f4-674739d43724.png)

Here you go @vomikan - UTF-8 was not checked: 

![image](https://user-images.githubusercontent.com/95182117/210255365-c0a3156a-599a-420b-952a-49afae811058.png)

And here is the version I had been testing with:

![image](https://user-images.githubusercontent.com/95182117/210255469-678813c3-5ed1-48d2-99c4-e77a404ce777.png)

thanks!

**Comment by vomikan** [comment]
> UTF-8 was not checked

If you enable and overload the OS, this should help.

**Comment by rmelillo76** [comment]
> > UTF-8 was not checked
> 
> 
> 
> If you enable and overload the OS, this should help.

Yes the arrow are now displayed correctly after enabling and an OS restart. I don't recall having changed that setting before, so I assume the default is to not be checked.  Should we reconsider how this change is being done, especially since Windows still refers to this as "beta" and many users would not have it enabled?

**Comment by n-stein** [comment]
> Should we reconsider how this change is being done, especially since Windows still refers to this as "beta" and many users would not have it enabled?

I agree, this solution seems sub-optimal. Do we really want to implement a change that will require many (most?) Windows users to manually enable a "beta" setting?

**Comment by n-stein** [comment]
Here is a clunky alternative that doesn't require any settings to be changed:

![image](https://user-images.githubusercontent.com/16599929/210406068-f9176dce-fe4c-48d4-9d9c-e4597c09e063.png)

**Comment by rmelillo76** [comment]
> Here is a clunky alternative that doesn't require any settings to be changed:
> 
> ![image](https://user-images.githubusercontent.com/16599929/210406068-f9176dce-fe4c-48d4-9d9c-e4597c09e063.png)

Hi @n-stein, what happens to the Unicode characters when the string is cast back to a std::string?
